nkms/crypto/signature.py

```diff
@@ -59,3 +61,63 @@ def __radd__(self, other):
     def __eq__(self, other):
         # TODO: Consider constant time
         return bytes(self) == bytes(other) or self._der_encoded_bytes() == other
+
+
+signature_splitter = BytestringSplitter(Signature)
+
+
+class SignatureStamp(object):
+    """
+    Can be called to sign something or used to express the signing public
+    key as bytes.
+    """
+
+    def __init__(self, signing_keypair):
+        self._sign = signing_keypair.sign
+        self._as_bytes = bytes(signing_keypair.pubkey)
+        self._as_umbral_pubkey = signing_keypair.pubkey
+
+    def __bytes__(self):
+        return self._as_bytes
+
+    def __call__(self, *args, **kwargs):
+        return self._sign(*args, **kwargs)
+
+    def __hash__(self):
+        return int.from_bytes(self, byteorder="big")
+
+    def __eq__(self, other):
+        return other == bytes(self)
+
+    def __add__(self, other):
+        return bytes(self) + other
+
+    def __radd__(self, other):
+        return other + bytes(self)
+
+    def __len__(self):
+        return len(bytes(self))
+
+    def __bool__(self):
+        return True
```

Comment:
I cannot tell whether or not this is True

Reply:
`return True if not KPrasch.can_tell_whether_this_is_True else True`